[Swift in WebKit] Use swift-format to lint WebKit code

rr-codes opened the pull request:
#### ae1d259cd69f42dc13a09fc1c6e127baa4ed5662
<pre>
[Swift in WebKit] Use swift-format to lint WebKit code
<a href="https://bugs.webkit.org/show_bug.cgi?id=293842">https://bugs.webkit.org/show_bug.cgi?id=293842</a>
<a href="https://rdar.apple.com/152347799">rdar://152347799</a>

Reviewed by Abrar Rahman Protyasha.

Work towards having consistently formatted and linted Swift code; lint the WebKit codebase
for improved correctness and readability.

This is excluding WebKitSwift for now.

* Source/WebKit/Shared/AuxiliaryProcessExtensions/GPUProcessExtension.swift:
* Source/WebKit/Shared/AuxiliaryProcessExtensions/NetworkingProcessExtension.swift:
* Source/WebKit/Shared/AuxiliaryProcessExtensions/WebContentProcessExtension.swift:
* Source/WebKit/UIProcess/API/Cocoa/ObjectiveCBlockConversions.swift:

- Replace block quotes.

* Source/WebKit/UIProcess/API/Cocoa/WKContextMenuElementInfoAdapter.swift:

- Ignore AllPublicDeclarationsHaveDocumentation due to SPI.

* Source/WebKit/UIProcess/API/Cocoa/WebKitSwiftOverlay.swift:

- Replace block quotes.
- Ignore AllPublicDeclarationsHaveDocumentation due to pre-existing API documentation.
- Use null coalescing instead of force-unwrapping
- Add missing documentation for some WKWebExtension initializers, based on their Objective-C documentation.
- Add missing documentation for `WKWebsiteDataStore.proxyConfigurations` based on its Objective-C documentation.

* Source/WebKit/UIProcess/API/Swift/URLSchemeHandler.swift:

- Add missing docs.

* Source/WebKit/UIProcess/API/Swift/WebPage+BackForwardList.swift:

- Add missing docs.

* Source/WebKit/UIProcess/API/Swift/WebPage+Configuration.swift:

- Add missing docs.
- Fix variable naming.

* Source/WebKit/UIProcess/API/Swift/WebPage+DialogPresenting.swift:

- Fix doc format.

* Source/WebKit/UIProcess/API/Swift/WebPage+Navigation.swift:

- Fix doc format.
- Ignore AllPublicDeclarationsHaveDocumentation due to SPI.

* Source/WebKit/UIProcess/API/Swift/WebPage+NavigationDeciding.swift:

- Ignore AllPublicDeclarationsHaveDocumentation due to SPI.
- Add a FIXME to SPI that should be API.

* Source/WebKit/UIProcess/API/Swift/WebPage+NavigationPreferences.swift:

- Fix variable naming

* Source/WebKit/UIProcess/API/Swift/WebPage.swift:

- Fix doc format
- Fix parameter and variable naming
- Add missing docs
- Ignore AllPublicDeclarationsHaveDocumentation in some places due to SPI.

* Source/WebKit/UIProcess/Cocoa/GroupActivities/WKGroupSession.swift:

- Fix variable naming
- Justify force-unwrapping

* Source/WebKit/UIProcess/Cocoa/RunLoopQueue.swift:

- Add documentation (even though this is SPI).

* Source/WebKit/UIProcess/Cocoa/TextExtraction/WKTextExtractionItem.swift:

- Fix variable naming.

* Source/WebKit/UIProcess/Cocoa/TextExtraction/WKWebView+TextExtraction.swift:

- Ignore NoLeadingUnderscores because it is a false-positive.

* Source/WebKit/UIProcess/Cocoa/WKNavigationDelegateAdapter.swift:

- Ignore NoLeadingUnderscores because it is a false-positive.

* Source/WebKit/UIProcess/Cocoa/WKScrollGeometryAdapter.swift:
* Source/WebKit/UIProcess/Cocoa/WKUIDelegateAdapter.swift:
* Source/WebKit/UIProcess/Cocoa/WebPageWebView.swift:

- Ignore AllPublicDeclarationsHaveDocumentation due to SPI.

Canonical link: <a href="https://commits.webkit.org/295655@main">https://commits.webkit.org/295655@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e653670990386ceca400294a9af2d94bee57ed36

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/105785 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/25536 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/15929 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/110982 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/56381 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/26058 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/34039 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/80335 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/108791 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/20414 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/95458 "Passed tests") | [❌ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/60648 "Found 143 new API test failures: /WPE/TestUIClient:/webkit/WebKitWebView/query-permission-requests, /WPE/TestWebsiteData:/webkit/WebKitWebsiteData/deviceidhashsalt, /WPE/TestLoaderClient:/webkit/WebKitWebView/load-twice-and-reload, /WPE/TestCookieManager:/webkit/WebKitCookieManager/ephemeral, /WPE/TestWebKitWebView:/webkit/WebKitWebView/is-playing-audio, /WPE/TestConsoleMessage:/webkit/WebKitConsoleMessage/network-error, /WPE/TestWebsiteData:/webkit/WebKitWebsiteData/configuration, /WPE/TestCookieManager:/webkit/WebKitCookieManager/delete-cookies, /WPE/TestWebKitWebView:/webkit/WebKitWebView/terminate-unresponsive-web-process, /WPE/TestSSL:/webkit/WebKitWebView/insecure-content ... (failure)") | 
| | [⏳ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/iOS-18-Simulator-WPT-WK2-Tests-EWS "Waiting to run tests") | | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/55820 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/89843 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/13594 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/113831 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/32925 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/24263 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/89414 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/33289 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/91689 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/89084 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/22704 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/33960 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/11763 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/28403 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/32850 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/38261 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/32596 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/35945 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/34194 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aprotyas commented on Source/WebKit/UIProcess/API/Cocoa/WebKitSwiftOverlay.swift:
```diff
@@ -120,11 +140,25 @@ extension WKWebView {
+    /// If the manifest is invalid or missing, or the URL points to an unsupported format or invalid archive, an error will be returned.
+    ///
+    /// - Parameter resourceBaseURL: The file URL to use for the new web extension.
+    /// - Throws: An error If the manifest is invalid or missing, or the URL points to an unsupported format or invalid archive.
```
nit
```suggestion
    /// - Throws: An error if the manifest is invalid or missing, or the URL points to an unsupported format or invalid archive.
```

aprotyas reviewed (approved):
Small thing to fix in the commit message

> Include a Radar link (OOPS!).

rr-codes committed:
[Swift in WebKit] Use swift-format to lint WebKit code
https://bugs.webkit.org/show_bug.cgi?id=293842
rdar://152347799

Reviewed by Abrar Rahman Protyasha.

Work towards having consistently formatted and linted Swift code; lint the WebKit codebase
for improved correctness and readability.

This is excluding WebKitSwift for now.

* Source/WebKit/Shared/AuxiliaryProcessExtensions/GPUProcessExtension.swift:
* Source/WebKit/Shared/AuxiliaryProcessExtensions/NetworkingProcessExtension.swift:
* Source/WebKit/Shared/AuxiliaryProcessExtensions/WebContentProcessExtension.swift:
* Source/WebKit/UIProcess/API/Cocoa/ObjectiveCBlockConversions.swift:

- Replace block quotes.

* Source/WebKit/UIProcess/API/Cocoa/WKContextMenuElementInfoAdapter.swift:

- Ignore AllPublicDeclarationsHaveDocumentation due to SPI.

* Source/WebKit/UIProcess/API/Cocoa/WebKitSwiftOverlay.swift:

- Replace block quotes.
- Ignore AllPublicDeclarationsHaveDocumentation due to pre-existing API documentation.
- Use null coalescing instead of force-unwrapping
- Add missing documentation for some WKWebExtension initializers, based on their Objective-C documentation.
- Add missing documentation for `WKWebsiteDataStore.proxyConfigurations` based on its Objective-C documentation.

* Source/WebKit/UIProcess/API/Swift/URLSchemeHandler.swift:

- Add missing docs.

* Source/WebKit/UIProcess/API/Swift/WebPage+BackForwardList.swift:

- Add missing docs.

* Source/WebKit/UIProcess/API/Swift/WebPage+Configuration.swift:

- Add missing docs.
- Fix variable naming.

* Source/WebKit/UIProcess/API/Swift/WebPage+DialogPresenting.swift:

- Fix doc format.

* Source/WebKit/UIProcess/API/Swift/WebPage+Navigation.swift:

- Fix doc format.
- Ignore AllPublicDeclarationsHaveDocumentation due to SPI.

* Source/WebKit/UIProcess/API/Swift/WebPage+NavigationDeciding.swift:

- Ignore AllPublicDeclarationsHaveDocumentation due to SPI.
- Add a FIXME to SPI that should be API.

* Source/WebKit/UIProcess/API/Swift/WebPage+NavigationPreferences.swift:

- Fix variable naming

* Source/WebKit/UIProcess/API/Swift/WebPage.swift:

- Fix doc format
- Fix parameter and variable naming
- Add missing docs
- Ignore AllPublicDeclarationsHaveDocumentation in some places due to SPI.

* Source/WebKit/UIProcess/Cocoa/GroupActivities/WKGroupSession.swift:

- Fix variable naming
- Justify force-unwrapping

* Source/WebKit/UIProcess/Cocoa/RunLoopQueue.swift:

- Add documentation (even though this is SPI).

* Source/WebKit/UIProcess/Cocoa/TextExtraction/WKTextExtractionItem.swift:

- Fix variable naming.

* Source/WebKit/UIProcess/Cocoa/TextExtraction/WKWebView+TextExtraction.swift:

- Ignore NoLeadingUnderscores because it is a false-positive.

* Source/WebKit/UIProcess/Cocoa/WKNavigationDelegateAdapter.swift:

- Ignore NoLeadingUnderscores because it is a false-positive.

* Source/WebKit/UIProcess/Cocoa/WKScrollGeometryAdapter.swift:
* Source/WebKit/UIProcess/Cocoa/WKUIDelegateAdapter.swift:
* Source/WebKit/UIProcess/Cocoa/WebPageWebView.swift:

- Ignore AllPublicDeclarationsHaveDocumentation due to SPI.

Canonical link: https://commits.webkit.org/295655@main
